lib/vrl/stdlib/src/mod.rs

```diff
@@ -0,0 +1,106 @@
+use ::value::Value;
+use vrl::prelude::*;
+
+fn r#mod(value: Value, modulus: Value) -> Resolved {
+    let result = value.try_rem(modulus)?;
+    Ok(result)
+}
+
+#[derive(Clone, Copy, Debug)]
+pub struct Mod;
+
+impl Function for Mod {
+    fn identifier(&self) -> &'static str {
+        "mod"
+    }
+
+    fn parameters(&self) -> &'static [Parameter] {
+        &[
+            Parameter {
+                keyword: "value",
+                kind: kind::ANY,
```

Comment:
This shouldn't be `any`. Same for `modulus`.

Reply:
I forgot we could combine these types (since both integer and float are valid).
Fixed.